hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/NNThroughputBenchmark.java

```diff
@@ -265,6 +265,11 @@ void benchmark() throws IOException {
         LOG.info("Starting " + numOpsRequired + " " + getOpName() + "(s).");
         for(StatsDaemon d : daemons)
           d.start();
+      } catch (Exception e) {
+        if (e instanceof ArrayIndexOutOfBoundsException) {
+          LOG.error("The -dirsPerDir or -filesPerDir parameter set is incorrect.");
```

Comment:
It would be really nice to suggest a valid range of number of directories or files. For example, up to 1 million directories?

Reply:
Thanks @jojochuang for the comment and review.
I agree with your suggestion, and I will update it later.